spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -73,12 +74,23 @@
       end
 
       context 'when both ipp and opt-in ipp are enabled' do
-        it 'renders the show template for how to verify' do
-          get :show
-
-          expect(Idv::HowToVerifyController.enabled?).to be true
-          expect(subject.idv_session.skip_doc_auth).to be_nil
-          expect(response).to render_template :show
+        context 'when the ServiceProvider has IPP enabled' do
```

Comment:
Need to add a `disabled` path as well.

I am trying to avoid the path of needing to test every permutation of IPP feature flag × opt-in feature flag × ServiceProvider attribute, because some are nonsensical. (But, all are theoretically possible.)